docs/onboarding/03-sdks-and-apis.md

```diff
@@ -0,0 +1,34 @@
+# Chapter 3: Digital Twin SDK's and API's
+
+Azure digital twins provides a C# SDK for programmatic management of models and twins. The following covers the capabilities of the SDK's and APIs

```

Comment:
ADT also offers a Java SDK. Perhaps worth mentioning, although you can always say "these onboarding documents all focus around features in C#, so bear that in mind if you are developing a Java project" if you want to make it clear we're not sure about feature parity

Reply:
Clarified this doc and added your suggestion to the root onboarding readme.